ament_cmake_auto/cmake/ament_auto_package.cmake

```diff
@@ -87,6 +90,20 @@ macro(ament_auto_package)
     )
   endif()
 
+  # install directories to share
+  foreach(_dir ${_ARG_INSTALL_TO_SHARE})
+    if(IS_ABSOLUTE "${_dir}")
+      get_filename_component(_dir_name "${_dir}" NAME)
+    else()
+      set(_dir_name "${_dir}")
```

Comment:
Doesn't this case also need `get_filename_component(_dir_name "${_dir}" NAME)`? Otherwise how can you install a directory `foo/bar` into `share/<pkgname>/bar`?

Reply:
I guess I assumed that if you are installing `foo/bar` it would end up in `share/<pkgname>/foo/bar`, but I also don't have a case where this is happening, so we simplify it for now (use `get_filename_component(_dir_name "${_dir}" NAME)` in all cases